zerver/lib/logging_util.py

```diff
@@ -177,11 +177,17 @@ class ZulipFormatter(logging.Formatter):
     # Used in the base implementation.  Default uses `,`.
     default_msec_format = '%s.%03d'
 
-    _fmt = '%(asctime)s %(zulip_level_abbrev)-4s [%(zulip_origin)s] %(message)s'
-
     def __init__(self):
         # type: () -> None
-        super().__init__(fmt=self._fmt)
+        super().__init__(fmt=self._fmt())
+
+    def _fmt(self):
+        # type: () -> str
+        pieces = ['%(asctime)s', '%(zulip_level_abbrev)-4s']
```

Comment:
There were lint errors .

Reply:
Yup, those are annoying -- added a commit to fix that rule in general.